Cakes-Co/style.css

```diff
@@ -1 +1,344 @@
+    grid-template-rows: repeat(2, 10rem);
+    row-gap: 2rem;
+    column-gap: 2rem;
+  }
+
+  /* FOOTER STYLES - A FEW LINKS AND SOCIAL MEDIA LINKS */
+
+  footer div:nth-child(1) {
+    display: flex;
+    gap: 0.4rem;
+    flex-direction: column;
+  }
+
+  footer {
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    align-items: center;
+    column-gap: 2rem;
+    width: 82%;
+  }
+
+  .socials-svgs {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 0.6rem;
+    text-align: end;
+  }
+}
+
+/*  */
+/*  */
+/*  */
+/*  */
+
+@media only screen and (min-width: 767px) and (max-width: 1650px) {
+  body {
+    padding: 0;
+    margin: 0;
+    position: relative;
+  }
+
+  .layout {
+    display: grid;
+    grid-template-rows: 250px 100px 500px 460px 350px;
+    row-gap: 3rem;
+  }
+
+  header {
+    background-color: var(--layout-color);
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    justify-content: space-between;
+    padding: 0 2%;
+    border-bottom-left-radius: 100%;
+  }
+
+  header p {
+    font-size: 2rem;
+    padding-bottom: 1rem;
+    align-self: end;
+    justify-self: end;
+  }
+
+  .cupcake-logo {
+    align-self: center;
+    border-radius: 100%;
+    margin-left: 1rem;
+  }
+
+  header svg {
+    display: none;
+  }
+
+  nav {
+    background-color: var(--background);
+    display: flex;
+    justify-content: end;
+    align-items: center;
+    padding-right: 2rem;
+    border-bottom: 2px solid var(--layout-color);
+  }
+
+  .flex-nav {
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+  }
+
+  .flex-nav a:hover,
+  .flex-nav a:active {
+    border-top-left-radius: 24px;
+    border-bottom-right-radius: 24px;
+    border-top-right-radius: 24px;
+    border-bottom-left-radius: 24px;
+    opacity: 1;
+  }
+
+  .flex-nav a {
+    display: flex;
+    color: white;
+    background-color: var(--brand);
+    padding: 0.8rem 1.4rem;
+    transition: border-top-left-radius ease-in-out 0.3s,
+      border-bottom-right-radius ease-in-out 0.6s,
+      border-top-right-radius ease-in-out 0.9s,
+      border-bottom-left-radius ease-in-out 1.2s, opacity ease-in-out 1.2s;
+    opacity: 0.6;
+  }
+
+  /* .flex-nav a:hover,
+  .flex-nav a:active {
+    border-radius: 24px;
+    opacity: 1;
+  } */
+
+  main {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    column-gap: 2rem;
+  }
+
+  main div:nth-child(2) h1 {
+    font-size: 3.9rem;
+    text-align: end;
+  }
+
+  main div:nth-child(2) p {
+    line-height: 1.2;
+    text-align: end;
+    font-size: 1.4rem;
+  }
+
+  content {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    column-gap: 2rem;
+  }
+
+  content div img {
+    border-radius: 10px;
+    opacity: 0.6;
+    transition: opacity var(--transition-effect);
+    cursor: pointer;
+  }
+
+  content div img:hover,
+  content div img:active {
+    opacity: 1;
+  }
+
+  /* content div:nth-child(1) {
```

Comment:
Don't leave commented out code in your PRs

Reply:
I'll resolve this now sorry Sally

Reply:
Now removed dead code